cms/search/serializers.py

```diff
@@ -0,0 +1,21 @@
+from typing import TYPE_CHECKING
+
+from rest_framework import serializers
+
+from cms.search.utils import build_resource_dict
+
+if TYPE_CHECKING:
+    from wagtail.models import Page
+
+
+class ResourceSerializer(serializers.Serializer):
```

Comment:
question: why not use `serializers.BaseSerializer`? https://www.django-rest-framework.org/api-guide/serializers/#read-only-baseserializer-classes recommnds using it for read-only cases, and you don't need to override `create`/`update`

Reply:
Done! Correct, remember reading this, lost it.